dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -95,6 +96,18 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
```

Comment:
Does the public WIL Tracelogging have "Activity" support? In that, you'd say:

```c++
auto registerActivity = AppNotificationTelemetry::RegisterStart(m_appId);
```

... and then the activity manages the start/stop.

Reply:
@loneursid Do you know? Added this telemetry point recently. Not aware if this support exists publicly.